src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: $normal-font;
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    font-weight: 500;
+  }
+
+  &__item {
+    margin-right: 20px;
+    position: relative;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    height: 60px;
+    line-height: 60px;
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
```

Comment:
have you read this comment?
https://github.com/mate-academy/layout_catalog-en/pull/16#discussion_r837843272

Reply:
I couldnt get it but now I know what you mean.